frontend/src/components/molecules/MatchItem/MatchItem.tsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react';
+import styled from "styled-components";
+import ListItemAvatar from '@material-ui/core/ListItemAvatar';
+import ListItemText from '@material-ui/core/ListItemText';
+import {Avatar, Card, Typography, withStyles} from '@material-ui/core';
+import {MatchTeam} from "../MatchTeam/MatchTeam";
+import Accordion from '@material-ui/core/Accordion';
+import AccordionDetails from '@material-ui/core/AccordionDetails';
+import AccordionSummary from '@material-ui/core/AccordionSummary';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import {THEME} from "../../atoms/constants/ThemeMUI";
+import AddCircleTwoToneIcon from '@material-ui/icons/AddCircleTwoTone';
+import MatchWinnerDeterminationDialog from "../MatchWinnerDeterminationDialog/MatchWinnerDeterminationDialog";
+
+export type MatchItemProps = {
+    onClickTeam: () => void,
+    matchNumber: number | undefined,
```

Comment:
Moze nie byc numeru meczu?

Reply:
Powinienem najpierw pobrać dane z bazy bo tu nie jestem pewien co z niej dostanę, czy string czy number

Reply:
hmmm czemu string? matchNumber jest typu number z tego co widzę w swaggerze

Reply:
masz rację, ogarnę to :)